source/version.js

```diff
@@ -55,3 +52,10 @@ module.exports.validate = version => {
 		throw new Error('Version should be a valid semver version.');
 	}
 };
+
+module.exports.satisfies = (version, range) => {
```

Comment:
Why did you add this?
Up until now we used `version(versionNumber).satisfies(range)`, which should still work perfectly fine.